contracts/token/FeiTimedMinter.sol

```diff
@@ -0,0 +1,105 @@
+pragma solidity ^0.8.0;
+
+import "../refs/CoreRef.sol";
+import "../utils/Timed.sol";
+import "../utils/Incentivized.sol";
+import "../utils/RateLimitedMinter.sol";
+import "./IFeiTimedMinter.sol";
+
+/// @title FeiTimedMinter
+/// @notice a contract which mints FEI to a target address on a timed cadence
+contract FeiTimedMinter is IFeiTimedMinter, CoreRef, Timed, Incentivized, RateLimitedMinter {
```

Comment:
This might be more for v3, but we should consider adding in a dynamic mint amount - based off of a curve or even current system conditions.

Reply:
Totally